profiler/src/ProfilerEngine/Datadog.Linux.ApiWrapper/functions_to_wrap.c

```diff
@@ -319,6 +322,9 @@ void initLibrary(void)
             setenv("DOTNET_DbgEnableMiniDump", "1", 1);
             setenv("DD_TRACE_CRASH_HANDLER_PASSTHROUGH", "0", 1);
         }
+
+        originalMiniDumpName = getenv("COMPlus_DbgMiniDumpName");
```

Comment:
The customer could have set `DOTNET_DbgMiniDumpName` so you have to check both (see how it was done for `COMPlus_DbgEnableMiniDump`/`DOTNET_DbgEnableMiniDump`)